src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -2888,8 +2888,8 @@ public void setStereoscopy(String stereoscopy) {
 		SBSL,
 		SBSR,
 		HSBSL,
-		OUL,
-		OUR,
+		ABL,
+		ABR,
 		HOUL,
```

Comment:
@Sami32 you are right I will also change it. The same apply to your comment above.